client/lib/plans/constants.js

```diff
@@ -548,10 +564,12 @@ export const PLANS_LIST = {
 		] ),
 		getSignupFeatures: () => compact( [
 			FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME,
-			FEATURE_ONE_CLICK_THREAT_RESOLUTION,
+			FEATURE_VIDEO_CDN_UNLIMITIED,
+			FEATURE_SEO_PREVIEW_TOOLS,
 			FEATURE_ALL_PREMIUM_FEATURES
 		] ),
-		getBillingTimeFrame: () => i18n.translate( 'per month, billed yearly' ),
+		getBillingTimeFrame: () => i18n.translate( 'per month, billed monthly' ),
+		getSignupBillingTimeFrame: () => 'per month', //PLANS A/B TEST: Translate if test passes
```

Comment:
Any good reason to not wrap this one in a `translate` call immediately? In the worst case we could lose several translated versions of it if we decide to not keep the test.